rrfs-test/validated_yamls/templates/obtype_config/amsua_n19.yaml

```diff
@@ -30,14 +30,14 @@
          - localization method: Horizontal Gaspari-Cohn
            lengthscale: 300e3 # orig
        obs bias:
-         input file: "@BIASFILE@"
+         input file: data/obs/amsua_n19.satbias.nc4
```

Comment:
Why the input bias file has a `.nc4` extension  while the output file has a `.nc` extension?
(the obsfile also has a `.nc` extension)
Is this because of current JEDI practice?  Thanks!

Reply:
I didn't pay attention to that. But I think they are same. I can change all 'nc' to 'nc4' for consistence.

Reply:
Thanks!